### PR TITLE
Repo refactor

### PR DESCRIPTION
**Pull request by martialblog**
Addresses Issue #102

### CODE REVIEW

**Comment by martialblog** [comment]
Ok the RuboCop Check failed... I'll have a look at that

**Comment by martialblog** [comment]
Just now rebased the branch

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -24,8 +24,8 @@
     puppetversion: Puppet.version,
     facterversion: Facter.version
   }
-  default_facts.merge!(YAML.load(File.read(File.expand_path('../default_facts.yml', __FILE__)))) if File.exist?(File.expand_path('../default_facts.yml', __FILE__))
-  default_facts.merge!(YAML.load(File.read(File.expand_path('../default_module_facts.yml', __FILE__)))) if File.exist?(File.expand_path('../default_module_facts.yml', __FILE__))
+  default_facts.merge!(YAML.safe_load(File.read(File.expand_path('../default_facts.yml', __FILE__)))) if File.exist?(File.expand_path('../default_facts.yml', __FILE__))
```
Comment (bastelfreak) [inline spec/spec_helper.rb]: please check this again. I think the rebase didn't work as expected. We should stay with YAML.load.

**Comment by bastelfreak** [comment]
Hi @martialblog, thanks for the PR! Can you please check the spec/spec_helper.rb again? After thats gone from the PR I'm happy to merge + release.

**Comment by martialblog** [comment]
Hey @bastelfreak, all done. Should look good now.

**Comment by bastelfreak** [comment]
thanks!

**Comment by bastelfreak** [comment]
Hi @martialblog, I just released a new version: https://forge.puppet.com/puppet/gluster

**Pull request merged**